src/libs/ReportUtils.js

```diff
@@ -751,6 +753,21 @@ function getAvatar(avatarURL, login) {
     return avatarURL;
 }
 
+/**
+ * Provided an avatar URL, if avatar is a default avatar, return NewDot default avatar URL.
+ * Otherwise, return the URL pointing to a user-uploaded avatar.
+ *
+ * @param {String} [avatarURL] - the avatar source from user's personalDetails
+ * @param {String} [login] - the email of the user
+ * @returns {String|Function}
```

Comment:
How does this return a function? According to JSDocs, `avatarURL` is a string and `getDefaultAvatarURL()` returns a string.